lib/go/test/examples.go

```diff
@@ -37,69 +38,72 @@ func DownloadFile(url string) ([]byte, error) {
 	return ioutil.ReadAll(resp.Body)
 }
 
-// newEmulator returns a emulator object for testing.
-func NewEmulator() *emulator.Blockchain {
-	b, err := emulator.NewBlockchain()
+// newBlockchain returns an emulator blockchain for testing.
+func newBlockchain(opts ...emulator.Option) *emulator.Blockchain {
+	b, err := emulator.NewBlockchain(
+		append(
+			[]emulator.Option{
+				emulator.WithStorageLimitEnabled(false),
+			},
+			opts...,
+		)...,
+	)
 	if err != nil {
 		panic(err)
 	}
 	return b
 }
 
-// createSignAndSubmit creates a new transaction and submits it
-func createSignAndSubmit(
-	t *testing.T,
+func createTxWithTemplateAndAuthorizer(
 	b *emulator.Blockchain,
-	template []byte,
-	signerAddresses []flow.Address,
-	signers []crypto.Signer,
-	shouldRevert bool,
-) {
+	script []byte,
+	authorizerAddress flow.Address,
+) *flow.Transaction {
 
 	tx := flow.NewTransaction().
-		SetScript(template).
+		SetScript(script).
 		SetGasLimit(9999).
 		SetProposalKey(b.ServiceKey().Address, b.ServiceKey().Index, b.ServiceKey().SequenceNumber).
-		SetPayer(b.ServiceKey().Address)
-	for _, addr := range signerAddresses[1:] {
-		tx = tx.AddAuthorizer(addr)
-	}
+		SetPayer(b.ServiceKey().Address).
+		AddAuthorizer(authorizerAddress)
 
-	SignAndSubmit(
-		t, b, tx,
-		signerAddresses,
-		signers,
-		shouldRevert,
-	)
+	return tx
 }
 
-// SignAndSubmit signs a transaction with an array of signers and adds their signatures to the transaction
-// Then submits the transaction to the emulator. If the private keys don't match up with the addresses,
-// the transaction will not succeed.
-// shouldRevert parameter indicates whether the transaction should fail or not
-// This function asserts the correct result and commits the block if it passed
-func SignAndSubmit(
+// signAndSubmit signs a transaction with an array of signers and adds their signatures to the transaction
+// before submitting it to the emulator.
+//
+// If the private keys do not match up with the addresses, the transaction will not succeed.
+//
+// The shouldRevert parameter indicates whether the transaction should fail or not.
+//
+// This function asserts the correct result and commits the block if it passed.
+func signAndSubmit(
```

Comment:
It'd be great to get these utilities into the `flow-go-sdk/test` package: https://github.com/onflow/flow-go-sdk/tree/master/test

Or maybe the emulator?

Reply:
Yeah that is a good idea. What would be the best way to do that? Can we just put them in as is, or should we have a discussion about which parts we want to include?